music_assistant/server/providers/snapcast/__init__.py

```diff
@@ -109,7 +109,14 @@ async def get_config_entries(
     values: the (intermediate) raw values for config entries sent with the action.
     """
     returncode, output = await check_output(["snapserver", "-v"])
-    snapserver_present = returncode == 0 and "snapserver v0.27.0" in output.decode()
+    snapserver_version: int = int(output.decode().split(".")[1]) if returncode == 0 else -1
+    snapserver_valid_version: bool = snapserver_version >= 27
+    snapserver_present = (
+        returncode == 0 and "snapserver" in output.decode() and snapserver_valid_version
+    )
```

Comment:
I can imagine this being obsolete now that you have an explicit version check ?

Reply:
Yes, you're right, in 5 minutes I'll change it.